docs/REST API/rest-api.md

```diff
@@ -8548,8 +8571,67 @@ Returns information about the specified software version.
 }
 ```
 
-## Vulnerabilities
+### Get available App Store apps
+
+Returns the list of App Store (VPP) apps purchased in Apple Business Manager.
+
+`GET /api/v1/fleet/software/app_store/:team_id`
+
+#### Parameters
+
+| Name    | Type | In | Description |
+| ------- | ---- | -- | ----------- |
+| team_id | integer | query | **Required**. The team ID. Lists available VPP software for specified team. |
+
+#### Example
+
+`GET /api/v1/fleet/software/app_store/3`
+
+##### Default response
 
+`Status: 200`
+
+```json
+{
+  "app_store_apps": {
+    {
+      "name": "Xcode",
+      "latest_version": "15.4",
+      "app_store_id": 497799835
+    },
+    {
+      "name": "Logic Pro",
+      "latest_version": "2.04",
+      "app_store_id": 634148309
+    },
```

Comment:
```suggestion
    {
      "name": "Xcode",
      "latest_version": "15.4",
      "app_store_id": 497799835,
      "added": true
    },
    {
      "name": "Logic Pro",
      "latest_version": "2.04",
      "app_store_id": 634148309,
      "added": false
    },
```

Reply:
Also I think this API call is missing the `pngs` you expect in the response to be able to show the icons in the UI list.

Reply:
> Also I think this API call is missing the `pngs` you expect in the response to be able to show the icons in the UI list.

I missed this, thanks